assets/css/editor.scss

```diff
@@ -85,3 +85,10 @@ svg.wc-block-editor-components-block-icon--sparkles path {
 		}
 	}
 }
+
+.theme-twentytwentytwo {
```

Comment:
Is it something specific twenty-twentytwo or did we test this with other themes as well? I would suggest avoiding theme specific code here as this could turn into a rabbit hole of patches down the line.

Reply:
I tested this problem against the following themes:

- [Twenty Twenty-Two](https://wordpress.org/themes/twentytwentytwo/)
- [Twenty Twenty-One](https://wordpress.org/themes/twentytwentyone/)
- [Twenty Twenty](https://wordpress.org/themes/twentytwenty/)
- [Twenty Nineteen](https://wordpress.org/themes/twentynineteen/)

The only theme that was affected by this problem was the Twenty Twenty-Two. I agree that resolving this issue in upstream would be better. I solved it here for two reasons:

1. In the past, I've seen various pushbacks from theme developers arguing that issues like this would have to be addressed within the corresponding plugin.
2. In the past, we also addressed problems of other core themes:

https://github.com/woocommerce/woocommerce-blocks/blob/15f7b50556e1f1bb7045d3d56abba78fb9d14470/assets/css/editor.scss#L77-L87

https://github.com/woocommerce/woocommerce-blocks/blob/15f7b50556e1f1bb7045d3d56abba78fb9d14470/assets/css/style.scss#L184-L191

That said, we can give it a try to get this issue solved in upstream.

Reply:
Apparently, there was already a core ticket, so I added a note via https://core.trac.wordpress.org/ticket/55839#comment:3.

Reply:
Ah good spot! So it looks like the issue affects not just our plugin but all images aligned centre.